indico/modules/events/registration/fields/simple.py

```diff
@@ -110,6 +121,9 @@ def modify_post_data(cls, post_data):
 class BooleanField(RegistrationFormBillableField):
     name = 'yes/no'
     wtf_field_class = IndicoRadioField
+    mapping = {None: '',
```

Comment:
If you do it on the class level, use `L_()` instead of `_()` (or just move it inside the method)